website/docs/r/databricks_virtual_network_peering.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: "Databricks"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_databricks_virtual_network_peering"
+description: |-
+  Manages a Databricks Virtual Network Peering
+---
+
+# azurerm_databricks_virtual_network_peering
+
+Manages a Databricks Virtual Network Peering
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "databricks-vnet-peering"
```

Comment:
minor but fwiw we call this `example-resources` in other examples:

```suggestion
  name     = "example-resources"
```

Reply:
Fixed.